helm/estafette-cloudflare-dns/templates/deployment.yaml

```diff
@@ -27,6 +27,7 @@ spec:
       annotations:
         prometheus.io/scrape: "true"
         prometheus.io/port: "9101"
+        checksum/secrets: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
```

Comment:
Does this get evaluated after the values are filled into the secret?

Reply:
yes, tested with different values using Helm template.

Reply:
Excellent, let's apply this to the other official Estafette Helm charts making use of secrets and configmaps as well (unless the application can deal with config changes, but none of the current 9 controllers do).